drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -326,18 +326,50 @@ static int sdw_select_row_col(struct sdw_bus *bus, int clk_freq)
 	return -EINVAL;
 }
 
+static bool is_all_peripherals_connected(struct sdw_master_runtime *m_rt, unsigned int lane)
```

Comment:
nit-pick: 'is' is singular, this sound weird in English.

Maybe 'check_all_peripherals_connected'

Also need to explain the purpose of this function. I think it's useful to determine if a stream can use Lane N in all peripherals used by the stream on a given link are connected on Lane N?

Reply:
> nit-pick: 'is' is singular, this sound weird in English.
> 
> Maybe 'check_all_peripherals_connected'
> 
> Also need to explain the purpose of this function. I think it's useful to determine if a stream can use Lane N in all peripherals used by the stream on a given link are connected on Lane N?

Updated.